### ISSUE TITLE
custom link colour and show/hide link

### ISSUE THREAD

**Issue by zedzedzed** [zedzedzed/table-of-contents-plus#34]
**Reported by michael on 3 Sep 2011 08:48 UTC**
When custom link and link hover colours are defined, it never is applied to the show/hide link at the top.

I think it should.

**Comment by zedzedzed** [comment]
**Comment by michael on 3 Sep 2011 08:52 UTC**
In [109]:
```CommitTicketReference repository="" revision="109"
closes #36
also makes restores minimum heading depth to 4 (from 3)
```

**Comment by zedzedzed** [comment]
**Modified by michael on 12 Sep 2011 05:25 UTC**
